library/translation.inc.php

```diff
@@ -263,6 +263,15 @@ function html_header_show() {
     //  We commented this out since we are now standardizing encoding
     //  in the globals.php file.
```

Comment:
sorry in advance for the stupid question :)
Will the full blown AM require the html_header_show() hook to bring things in or will it use another mechanism?